features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -147,6 +147,13 @@
     <configfile finalname="${openhab.conf}/services/caldavPersonal.cfg" override="false">mvn:${project.groupId}/openhab-addons-external/${project.version}/cfg/caldavPersonal</configfile>
   </feature>
 
+  <feature name="openhab-binding-cardio2e" description="Cardio IIé binding" version="${project.version}">
```

Comment:
Per the convention, the name here should be `openhab-binding-cardio2e1`.

Reply:
Requested change is made.